Hover Ability Added

SirTangoGibus opened the pull request:
feat: Hover ability added as separate script. Can be toggled on/off within inspector and has multiple values that can be changed for functionality.
refactor: Made some variables within PlayerController read only for other scripts, changed some comment typos and empty spaces.

SirTangoGibus committed:
Hover Ability Added

feat: Hover ability added as separate script. Can be toggled on/off within inspector and has multiple values that can be changed for functionality.
refactor: Made some variables within PlayerController read only for other scripts, changed some comment typos and empty spaces.

MGHMSJ5 reviewed (approved):
Looks really nice! Really curious how this will work with the gravity

SirTangoGibus committed:
Pulse Ability Added

feat: base Pulse logic implemented.
Support for triggering platforms added.
Two types of platform logic added (vertical and horizontal movement).
Cable logic added (default and activated states).
Added a graybox playground Scene for abilities.
Added a few prototype materials to test out Pulse.
Added Pulse script to Player prefab.

SirTangoGibus commented:
Branch updated with Pulse:

feat: base Pulse logic implemented.
Support for triggering platforms added.
Two types of platform logic added (vertical and horizontal movement).
Cable logic added (default and activated states).
Added a graybox playground Scene for abilities.
Added a few prototype materials to test out Pulse.
Added Pulse script to Player prefab.
